### ISSUE TITLE
fluent-cat: uninitialized constant Fluent::Engine

### ISSUE THREAD

**Issue by soylent** [fluent/fluentd#746]
I got the following error:

```
$ cat fluentd_event.msgpack | fluent-cat --msgpack -u -s /tmp/fluent.sock test
uninitialized constant Fluent::Engine
```

It looks like `fluent/command/cat` is missing `require 'fluent/engine'`

**Comment by repeatedly** [comment]
What the version do you use?

**Comment by repeatedly** [comment]
Ah, okay. You are right.

**Comment by repeatedly** [comment]
fixed

**Comment by soylent** [comment]
Thank you!

**Comment by repeatedly** [comment]
Released v0.12.19 including this fix.
